tests/virt_autotest/login_console.pm

```diff
@@ -102,8 +102,8 @@ sub login_to_console {
         check_screen([qw(grub2 grub1)], 60);
     }
 
-    if (!get_var("reboot_for_upgrade_step")) {
-        set_var("first_reboot_after_upgrade", "no") if (check_var("first_reboot_after_upgrade", "yes"));
+    if (!get_var('UPGRADE_AFTER_REBOOT')) {
+        set_var('REBOOT_AFTER_UPGRADE', '') if (get_var('REBOOT_AFTER_UPGRADE'));
```

Comment:
How the var names explain themselves differ among readers, unless they are quite strightforward or simple. I guess most people can directly get 'OFFLINE_UPGRADE', while the other two vars may not be so easy. To avoid misunderstanding or confusion, maybe adding a comment explaining the var can help.

Reply:
Done